### PR TITLE
Precise that the `SafeMaskMaker` uses the true energy range to select events for the methods "aeff*"

### PR DESCRIPTION
**Pull request by bkhelifi**
**Description**

This pull request is associated to the issue #3708. It completes the documentation to precise the made assumption.

### CODE REVIEW

**Review by adonath — DISMISSED** [review]
Thanks @bkhelifi, looks good to me!

File: gammapy/makers/safe.py [hunk 1]
```diff
@@ -17,6 +17,11 @@
 class SafeMaskMaker(Maker):
     """Make safe data range mask for a given observation.
 
+    
+    .. warning::
+
```
Comment (registerrier) [inline gammapy/makers/safe.py:22]: Technically this is true for `aeff-default` `aeff-max` and `edisp-bias`as well.

Maybe :  'Note that currently some methods computing a safe energy range (`aeff-default` `aeff-max` and `edisp-bias`) determine a true energy range and apply it to reconstructed energy, effectively neglecting the energy bias.

File: examples/tutorials/api/makers.py [hunk 2]
```diff
@@ -131,7 +131,11 @@
 #    energy bin with the highest predicted background rate. This method
 #    was introduced in the HESS DL3 validation paper:
 #    https://arxiv.org/pdf/1910.08088.pdf
-# 
+#
```
Comment (registerrier) [inline examples/tutorials/api/makers.py:134]: See comment below

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi , a small clarification below

File: gammapy/makers/safe.py (hunk 1 above)
Comment (maxnoe) [reply gammapy/makers/safe.py:22]: energy bias → energy dispersion (it's not only the mean bias that is neglected, but the error in each individual event)

Comment (bkhelifi) [reply gammapy/makers/safe.py:22]: Indeed, both rms and bias play a role. And 'energy dispersion' is the term used in our IRF theory. Thanks @maxnoe

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi !

**Review by adonath — APPROVED** [review]

**Pull request merged**